Added examples where Switch as X can be used

dannytsang opened the pull request:
## Proposed change
Added example where Switch as X can be used outside of the integrations page.

## Type of change
<!--
    What types of changes does your PR introduce to our documentation/website?
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR.
-->

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information
<!--
    Details are important, and help maintainers processing your PR.
    Please be sure to fill out additional details, if applicable.
-->

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #23452

## Checklist
<!--
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR. If you're unsure about any of them, don't hesitate to ask.
    We're here to help! This is simply a reminder of what we are going to look
    for before merging your code.
-->

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

dannytsang committed:
Added examples where you can use the Switch as X functionality.

frenck commented on source/_integrations/switch_as_x.markdown:
```diff
@@ -43,3 +43,16 @@ Switch entity:
+ - [Helper](#helper)
+ - [Entity Properties](#entity-properties)
+
+### Helper
+{% my helpers title="Settings -> Devices & Services -> Helpers" %}
+![Helper example](/images/integrations/switch_as_x/Helper_Switch_as_X.png)
```
I'm not sure about this, this is already provided by the steps above?

![image](https://user-images.githubusercontent.com/195327/182315806-ccda3fcf-37c4-4926-90b0-74912088bece.png)

dannytsang replied:
Sorry my bad. I will remove it.

frenck commented on source/_integrations/switch_as_x.markdown:
```diff
@@ -43,3 +43,16 @@ Switch entity:
+{% my helpers title="Settings -> Devices & Services -> Helpers" %}
+![Helper example](/images/integrations/switch_as_x/Helper_Switch_as_X.png)
+
+### Entity Properties
+{% my entities title="Settings -> Devices & Services -> Entities" %}
+![Entity properties exampl](/images/integrations/switch_as_x/Entities_Properties.png)
```
Let's just make a single chapter on: Configuring switch as X from an existing entity as an alternative method or something (so we don't need all these paragraphs at all).

dannytsang replied:
I was struggling to fit the screenshots to the other ways of doing it but I will take another look later.

dannytsang replied:
Re-wrote this part along with the above comment to **Switch as X from existing entity**

dannytsang committed:
Addressed feedback from @frenck.

Removed Helper paragraph because it's covered in the manual configuration steps.

Re-wrote the entity properties where Switch as X can appear.

frenck commented on source/_integrations/switch_as_x.markdown:
```diff
@@ -43,3 +43,7 @@ Switch entity:
   description: The type of entity you want the switch to become.
 {% endconfiguration_basic %}
+
+## Switch as X from existing entity
+Where compatible, Switch as X will appear on entity properties. A list of entities can be found in {% my entities title="Settings -> Devices & Services -> Entities" %}.
+![Entity properties exampl](/images/integrations/switch_as_x/Entities_Properties.png)
```
```suggestion
## Switch as X from the existing entity

Where compatible, Switch as X will appear on entity properties. A list of entities can be found in {% my entities title="Settings -> Devices & Services -> Entities" %}.

![Entity properties exampl](/images/integrations/switch_as_x/Entities_Properties.png)
```

frenck committed:
Tiny tweak

frenck reviewed (approved):
Thanks, @dannytsang 👍
